ubidots/ubidots_in.html

```diff
@@ -1,111 +1,608 @@
+          </label>
+          <span  style="width: calc(15%)">
+         <button class="red-ui-button" id="ubidots-add-variable-button"><i class="fa fa-plus"></i></button>
+      </span>
 
+      </div>
+</script>
 
 <script type="text/x-red" data-help-name="ubidots_in">
-    <p>Fetch the last value from Ubidots and send it as a message to other nodes.</p>
-
-    <p>
-    Uses the <a href="https://github.com/mqttjs/MQTT.js" target="_blank">MQTT</a> to make the connection and suscribe
-    to the channel.
-    </p>
-
-    <p>
-    You must configure your node by double clicking it and filling the following information, all fields are required:
-    <br />
-
-    <ul>
-        <li>
-            <b>Account type</b> - The type of your account, whether it is a Ubidots or Ubidots for
-            Education account.
-        </li>
-        <li>
-            <b>Name</b> - The name to identify your MQTT client.
-        </li>
-        <li>
-            <b>Token</b> - Your Ubidots' account token. You can find it by following
-            <a href="http://help.ubidots.com/user-guides/find-your-token-from-your-ubidots-account" target="_blank">
-                this tutorial
-            </a>.
-        </li>
-        <li>
-            <b>Device label</b> - The label of the Device that contains the Variable where you want to fetch the values.
-        </li>
-        <li>
-            <b>Variable label</b> - The label of the Variable that you want to fetch the values from.
-        </li>
-    </ul>
-    </p>
-
-    <p>
-    Refer to the <a href="https://ubidots.com/docs/api/mqtt.html" target="_blank">Ubidots documentation</a> for more
-    information about the functionality of the MQTT broker.
-    </p>
+  <p>Subscribes to up to 10 variables or topics from Ubidots and sends them as a message to other nodes.</p>
+
+  <p>
+      Uses the <a href="https://github.com/mqttjs/MQTT.js" target="_blank">MQTT</a> library to establish the connection and suscribe to the topic.
+  </p>
+
+  <p>
+      Double click on the node to configure all the required fields.
+
+
+      <ul>
+          <li>
+            <p>
+              <b> Account Type: </b>Defaults to "Ubidots" (standard account). If you have an education account choose "Ubidots for Education".
+            </p>
+          </li>
+          <li>
+
+            <p>
+              <b> Name: </b> Label of node in Node-Red workspace. If empty, defaults to :"Ubidots in".
+            </p>
+          </li>
+          <li>
+            <p>
+              <b> Token: </b> (required) - Token necessary to authenticate the connection to your account. To obtain your token, login on <a href="https://www.ubidots.com" target:"_blank">ubidots.com</a>, under "My Profile" click on "API Credentials". <a href="https://help.ubidots.com/en/articles/590078-find-your-token-from-your-ubidots-account" target="_blank">Tutorial</a>
```

Comment:
Do not set this `(required)` label on this field, as every other field here (Account Type, Name and Device Label) are also required but they don't have the `(required)` label set explicitly.